.github/renovate.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": ["config:base"],
+  "labels": ["dependencies"],
+
+  "packageRules": [
+    {
+      "enabled": true,
+      "matchManagers": ["regex"],
+      "matchUpdateTypes": ["patch", "minor"],
+      "automerge": true
+    }
+  ],
+  "regexManagers": [
+    {
+      "fileMatch": ["renovate-regex.json$"],
+      "matchStrings": [
+        "\"renovate_datasource\": \"(?<datasource>.*?)\",\\s    \"renovate_depname\": \"(?<depName>.*?)\",\\s(    \"renovate_versioning\": \"(?<versioning>.*?)\",\\s)?(    \"sha256\": \"(?<currentDigest>.*?)\",\\s)?    \"version\": \"(?<currentValue>.*?)\""
+      ],
+      "versioningTemplate": "{{#if versioning}}{{{versioning}}}{{else}}semver-coerced{{/if}}"
+    }
+  ],
+  "reviewers": ["@chickenandpork"]
```

Comment:
Flags me as reviewer. I see them faster and can click "yes, machine".  Can still require clean checks at repo and org level.

Reply:
We can also flag a "team" within the "organization" as reviewers if we prefer.  As well, Renovate will happily "choose 2 at random from this team" (I think I had that working at an employer a year ago)

Reply:
sure why not a team?

Reply:
89fe6e7 implements a suggestion